cassandra-scheduler/src/main/java/com/mesosphere/dcos/cassandra/scheduler/seeds/SeedsManager.java

```diff
@@ -159,8 +160,16 @@ public SeedsManager(final DefaultConfigurationManager configurationManager,
         final List<String> seeds = new ArrayList<>(active.size());
 
         for (int seed = 0; seed < seedCount && seed < active.size(); ++seed) {
-            seeds.add(InetAddress.getByName(active.get(seed).getHostname())
-                    .getHostAddress());
+            CassandraDaemonTask daemon = active.get(seed);
+            Protos.ContainerStatus status = daemon.getTaskStatus().getContainerStatus();
```

Comment:
I believe with my changes to `CassandraState` that there will always be a `ContainerStatus` now since the `TaskStatus` is always available from `CassandraDaemonTask` which wasn't true before.